code/modules/mob/living/simple_animal/abnormality/waw/rose_sign.dm

```diff
@@ -410,9 +428,24 @@
 			sin = "death"
 
 /obj/structure/rose_work/proc/RoseLines(mob/living/simple_animal/hostile/abnormality/rose_sign/master)
-	var/list/speech_styles = list(SPAN_ITALICS, SPAN_SANS, SPAN_ROBOT, SPAN_ITALICS, SPAN_PAPYRUS, SPAN_SANS, SPAN_SINGING)
-	var/list/lines = list("Ah, how pretty.", "And so vivid, too!", "Your sin was such a beautiful hue of [sin].", "You've really made a fine addition to the garden.", \
-	"The color [sin]?! I'll plant it right next to me.", "Then... Shall we play some more?", "This garden will become terribly beautiful with more sinful flowers we bloom!")
+	var/list/speech_styles = list(
+		SPAN_ITALICS,
```

Comment:
This breaks rose sign's speech gimmick, which uses spans in a particular order

Reply:
gotcha, changed dat